src/RadixDlt.NetworkGateway.PostgresIntegration/Queries/ResourceSupplyQuery.cs

```diff
@@ -63,49 +63,52 @@
  */
 
 using Microsoft.EntityFrameworkCore;
-using RadixDlt.NetworkGateway.Abstractions;
-using RadixDlt.NetworkGateway.GatewayApi.Exceptions;
 using RadixDlt.NetworkGateway.PostgresIntegration.Models;
 using System.Collections.Generic;
+using System.Collections.Immutable;
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
 using GatewayModel = RadixDlt.NetworkGateway.GatewayApiSdk.Model;
 
-namespace RadixDlt.NetworkGateway.PostgresIntegration.Services;
+namespace RadixDlt.NetworkGateway.PostgresIntegration.Queries;
 
-internal static class QueryHelper
+internal static class ResourceSupplyQuery
```

Comment:
According to the naming convention, this would be `ResourceSupplyMultiLookupQuery`